user_guide_src/source/libraries/email.rst

```diff
@@ -113,35 +113,37 @@ sending email.
 =================== =================== ============================ =======================================================================
 Preference          Default Value       Options                      Description
 =================== =================== ============================ =======================================================================
-**userAgent**       CodeIgniter         None                         The "user agent".
-**protocol**        mail                ``mail``, ``sendmail``,      The mail sending protocol.
+**$fromEmail**                                                       The email address to be set in the "from" header.
+**$fromName**                                                        The name to be set in the "from" header. 
+**$userAgent**      CodeIgniter                                      The "user agent".
+**$protocol**       mail                ``mail``, ``sendmail``,      The mail sending protocol.
```

Comment:
Please remove all `$` marks in `Preference`.
Because these items may be keys in an array.